scripts/ci/publish.ts

```diff
@@ -657,9 +657,7 @@ Check them out at https://github.com/prisma/ecosystem-tests/actions?query=workfl
 async function getEnginesCommitHash(): Promise<string> {
   const prismaPath = path.resolve(process.cwd(), './packages/engines/package.json')
   const pkg = JSON.parse(await fs.promises.readFile(prismaPath, 'utf-8'))
-  // const engineVersion = pkg.prisma.version
-  const engineVersion = pkg.devDependencies['@prisma/engines-version']?.split('.').slice(-1)[0]
-
+  const engineVersion = pkg.devDependencies['@prisma/engines-version'].prisma.enginesVersion
```

Comment:
I don't think this property exists in `./packages/engines/package.json`. I'd suggest using a direct import from `import { prisma } from "@prisma/engines-version"` as suggested [here](https://github.com/prisma/prisma/pull/21653/files#r1374771640) or some similar approach.